picard/ui/itemviews.py

```diff
@@ -1158,3 +1175,24 @@ def decide_fingerprint_icon_info(file):
             icon = QtGui.QIcon()
             tooltip = _('No fingerprint was calculated for this file, use "Scan" or "Generate AcoustID Fingerprints" to calculate the fingerprint.')
         return (icon, tooltip)
+
+    @staticmethod
+    def decide_ab_icon_info(file):
+        if file.acousticbrainz_error:
+            icon = TrackItem.icon_error
+            tooltip = _('Extraction or submission of Acoustic features failed')
+            return (icon, tooltip)
```

Comment:
Since the method ends with `return (icon, tooltip)`, what about:

```python
        if file.acousticbrainz_error:
            icon = TrackItem.icon_error
            tooltip = _('Extraction or submission of Acoustic features failed')
        elif file.acousticbrainz_is_duplicate:
```

Reply:
Done.